htsohm/htsohm.py

```diff
@@ -201,10 +201,11 @@ def run_all_simulations(material):
     print('  PROPERTY\t|  VALUE\t|  BIN\t      |')
     print('----------------+---------------+-------------|')
 
-    simulation_types = config['simulations']
-    for simulation_type in simulation_types:
-        results = get_simulation(simulation_type).run(material, simulation_types[simulation_type])
-        material.update_from_dict(results)
+    simulation_config = config['simulations']
+    for simulation_type in simulation_config:
+        if simulation_type != 'directory':
+            results = get_simulation(simulation_type).run(material, simulation_config)
```

Comment:
I'm pretty sure this code doesn't run as-is. Did you miss a config? 

simulation_config refers to config['simulations'] but in your run methods you are referring to it as if it refers to 
config['simulations']['helium_void_fraction'] or the other individual tests.